pkg/notify/notify_test.go

```diff
@@ -12,28 +11,31 @@ type notifySuite struct{}
 var _ = check.Suite(&notifySuite{})
```

Comment:
find the test suite not really runs by registering `check.TestingT(t)`

Reply:
fixed